app/views/proprietor/users/new.html.erb

```diff
@@ -12,9 +12,8 @@
                 <%= pluralize(@user.errors.count, "error") %> prohibited this repository from being saved:
                 <ul>
                   <% @user.errors.messages.each do |key, messages| %>
-                  <% if @user.errors.details[key].present? %>
-                    <li><%= key %> &quot;<%= @user.errors.details[key].first[:value] %>&quot; <%= messages.join(' and ') %></li>
-                  <% end %>
+                    <% next unless messages.present? %>
+                    <li><%= key %> <%= messages.join(' and ') %></li>
```

Comment:
removed `@user.errors.details[key].first[:value]` because it doesn't exist per the screenshots below. unsure if it is supposed to or not.

if `[:value]` is supposed to exist, we should instead fix why it isn't showing up in the `details` array.

Reply:
### user "new" view page
![Screen Shot 2022-08-17 at 10 36 20 AM](https://user-images.githubusercontent.com/29032869/185200540-b3f794f7-08a2-45c4-bc6f-3a6e75c1c545.png)

### user controller
![Screen Shot 2022-08-17 at 10 47 32 AM](https://user-images.githubusercontent.com/29032869/185200604-246d36f8-9eab-472f-80bf-f541deb527b1.png)